src/Campaigns/resources/admin/components/FormModal/FormModal.module.scss

```diff
@@ -41,8 +41,9 @@
 
                 span:not(.givewp-field-required) {
                     color: var(--givewp-grey-400);
-                    font-size: 0.75rem;
+                    font-size: 0.875rem;
                     line-height: 1.5;
+                    margin-bottom: 0.4rem;
```

Comment:
@glaubersilva Why aren't we using the design system for these?

Reply:
@JasonTheAdams We don't have a var with this specific font size in our design system, but the design on Zeplin requires it.  About the spacing, I refactored it to use the `--givewp-spacing-2` var which has `0.5rem` as the value, which is the same used in the Zeplin design.